installer/lib/phx_new/generator.ex

```diff
@@ -95,4 +88,187 @@ defmodule Mix.Tasks.Phx.New.Generator do
+  end
+  defp get_ecto_adapter("postgres", app, module) do
+    {:postgrex, Ecto.Adapters.Postgres, db_config(app, module, "postgres", "postgres")}
+  end
+  defp get_ecto_adapter("sqlite", app, module) do
+    {:sqlite_ecto, Sqlite.Ecto,
+     dev:  [database: "db/#{app}_dev.sqlite"],
+     test: [database: "db/#{app}_test.sqlite", pool: Ecto.Adapters.SQL.Sandbox],
+     prod: [database: "db/#{app}_prod.sqlite"],
+     test_setup_all: "Ecto.Adapters.SQL.Sandbox.mode(#{inspect module}.Repo, :manual)",
+     test_setup: ":ok = Ecto.Adapters.SQL.Sandbox.checkout(#{inspect module}.Repo)",
+     test_async: "Ecto.Adapters.SQL.Sandbox.mode(#{inspect module}.Repo, {:shared, self()})"}
+  end
+  defp get_ecto_adapter("mongodb", app, module) do
+    {:mongodb_ecto, Mongo.Ecto,
+     dev:  [database: "#{app}_dev"],
+     test: [database: "#{app}_test", pool_size: 1],
+     prod: [database: "#{app}_prod"],
+     test_setup_all: "",
+     test_setup: "",
+     test_async: "Mongo.Ecto.truncate(#{inspect module}.Repo, [])",
+     binary_id: true,
+     migration: false,
+     sample_binary_id: "111111111111111111111111"}
+  end
+  defp get_ecto_adapter(db, _app, _mod) do
+    Mix.raise "Unknown database #{inspect db}"
+  end
+
+  defp db_config(app, module, user, pass) do
+    [dev:  [username: user, password: pass, database: "#{app}_dev", hostname: "localhost"],
+     test: [username: user, password: pass, database: "#{app}_test", hostname: "localhost",
+            pool: Ecto.Adapters.SQL.Sandbox],
+     prod: [username: user, password: pass, database: "#{app}_prod"],
+     test_setup_all: "Ecto.Adapters.SQL.Sandbox.mode(#{inspect module}.Repo, :manual)",
+     test_setup: ":ok = Ecto.Adapters.SQL.Sandbox.checkout(#{inspect module}.Repo)",
+     test_async: "Ecto.Adapters.SQL.Sandbox.mode(#{inspect module}.Repo, {:shared, self()})"]
+  end
+
+  defp kw_to_config(kw) do
+    Enum.map(kw, fn {k, v} ->
+      ",\n  #{k}: #{inspect v}"
+    end)
+  end
+
+  defp generator_config(adapter_config) do
+    adapter_config
+    |> Keyword.take([:binary_id, :migration, :sample_binary_id])
+    |> Enum.filter(fn {_, value} -> not is_nil(value) end)
+    |> case do
+      [] -> nil
+      conf ->
+        """
+
+        # Configure phoenix generators
+        config :phoenix, :generators#{kw_to_config(conf)}
+        """
+    end
+  end
+  defp phoenix_path(path, true) do
+    absolute = Path.expand(path)
+    relative = Path.relative_to(absolute, @phoenix)
+
+    if absolute == relative do
+      Mix.raise "--dev projects must be generated inside Phoenix directory"
+    end
+
+    relative
+    |> Path.split()
+    |> Enum.map(fn _ -> ".." end)
+    |> Path.join()
+  end
+
+  defp phoenix_path(_path, false) do
+    "deps/phoenix"
+  end
+
+  defp phoenix_dep("deps/phoenix"), do: ~s[{:phoenix, "~> 1.2.0"}]
+  # defp phoenix_dep("deps/phoenix"), do: ~s[{:phoenix, github: "phoenixframework/phoenix", override: true}]
```

Comment:
indeed. Right now it should point at this PR branch. At the moment you have to manually update the generated project and point it, but I'll bump the dep to GitHub for now